### PR TITLE
Updated release versions to 4.57.0 for azure-cosmos

### PR DESCRIPTION
**Pull request by kushagraThapar**
Updated release versions to 4.57.0 for azure-cosmos

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
-### 4.57.0-beta.1 (Unreleased)
+### 4.57.0 (2024-03-25)
 
 #### Features Added
 * Added public APIs `setMaxMicroBatchSize` and `getMaxMicroBatchSize` in `CosmosBulkExecutionOptions` - See [PR 39335](https://github.com/Azure/azure-sdk-for-java/pull/39335)
 
-#### Breaking Changes
-
 #### Bugs Fixed
 * Suppress exceptions when calling diagnostics handlers. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:9]: Suppressed

Comment (FabianMeiswinkel) [reply sdk/cosmos/azure-cosmos/CHANGELOG.md:9]: Also below `Only call System.exit` - should maybe be Changed logic to only call ,,,

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
-
 #### Bugs Fixed
-* Suppress exceptions when calling diagnostics handlers. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
+* Suppressed exceptions when calling diagnostics handlers. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
 * Fixed an issue where no cross region retry for write operations due to channel acquisition timeout. - See [PR 39255](https://github.com/Azure/azure-sdk-for-java/pull/39255)
 * Fixed incorrect container tag value in metrics. - See [PR 39322](https://github.com/Azure/azure-sdk-for-java/pull/39322)
 * Fixed issue where CosmosDiagnosticsContext is null when diagnostics are sampled out. - See [PR 39352](https://github.com/Azure/azure-sdk-for-java/pull/39352)
 #### Other Changes
-* Only call System.exit in `DiagnosticsProvider` for `Error` scenario. Also add `System.err` for `Error` cases. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
+* Changed logic to only call `System.exit()` in `DiagnosticsProvider` for `Error` scenario. Also add `System.err` for `Error` cases. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:14]:
```suggestion
* Changed logic to only call `System.exit()` in `DiagnosticsProvider` for `Error` scenario. Also added `System.err` for `Error` cases. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
```

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 3]
```diff
@@ -24,29 +22,29 @@
 
 #### Bugs Fixed
 * Fixed an issue where `sampleDiagnostics` is not being honored for `query. See [PR 37015](https://github.com/Azure/azure-sdk-for-java/pull/37015)
-* Suppress exceptions when calling diagnostics handlers. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
+* Suppressed exceptions when calling diagnostics handlers. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
 
 ### Other Changes
-* Only call `System.exit()` in `DiagnosticsProvider` for `Error` scenario. Also add `System.err` for `Error` cases. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
+* Changed logic to only call `System.exit()` in `DiagnosticsProvider` for `Error` scenario. Also add `System.err` for `Error` cases. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:28]:
```suggestion
* Changed logic to only call `System.exit()` in `DiagnosticsProvider` for `Error` scenario. Also added `System.err` for `Error` cases. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
```

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 4]
```diff
@@ -24,29 +22,29 @@
...
 ### 4.45.3-hotfix (2024-03-15)
 
 #### Bugs Fixed
 * Fixed an issue where `sampleDiagnostics` is not being honored for `query. See [PR 37015](https://github.com/Azure/azure-sdk-for-java/pull/37015)
-* Suppress exceptions when calling diagnostics handlers. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
+* Suppressed exceptions when calling diagnostics handlers. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
 
 ### Other Changes
-* Only call `System.exit()` in `DiagnosticsProvider` for `Error` scenario. Also add `System.err` for `Error` cases. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
+* Changed logic to only call `System.exit()` in `DiagnosticsProvider` for `Error` scenario. Also add `System.err` for `Error` cases. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:38]:
```suggestion
* Changed logic to only call `System.exit()` in `DiagnosticsProvider` for `Error` scenario. Also added `System.err` for `Error` cases. - See [PR 39077](https://github.com/Azure/azure-sdk-for-java/pull/39077)
```

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 5]
```diff
@@ -24,29 +22,29 @@
...
 
 ### 4.53.3-hotfix (2024-03-07)
 
 #### Bugs Fixed
-* Suppress exceptions when calling diagnostics handlers. - See [PR 39121](https://github.com/Azure/azure-sdk-for-java/pull/39121)
+* Suppressed exceptions when calling diagnostics handlers. - See [PR 39121](https://github.com/Azure/azure-sdk-for-java/pull/39121)
 
 #### Other Changes
-* Only call System.exit in DiagnosticsProvider for Error scenario. Also add System.err for Error cases. - - See [PR 39121](https://github.com/Azure/azure-sdk-for-java/pull/39121)
+* Changed logic to only call `System.exit()` in `DiagnosticsProvider` for `Error` scenario. Also add `System.err` for `Error` cases. - See [PR 39121](https://github.com/Azure/azure-sdk-for-java/pull/39121)
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:47]:
```suggestion
* Changed logic to only call `System.exit()` in `DiagnosticsProvider` for `Error` scenario. Also added `System.err` for `Error` cases. - See [PR 39121](https://github.com/Azure/azure-sdk-for-java/pull/39121)
```

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

**Review by jeet1995 — APPROVED** [review]
LGTM - thanks!

**Pull request merged**